baker_wet_cleaning_application/scripts/readme.txt

```diff
@@ -1,177 +0,0 @@
-================================================================================================
```

Comment:
Why was the readme removed? I would leave it.

Reply:
Fixed. I updated the readme and renamed it `readme.md` to support markdown.